mozilla/blurts-server
# L10 hook hydration error

# Description

This PR fixes the l10 hook hydration errors, adds some missing strings, and addresses some warnings.

# How to test

Visit the new dashboard `/redesign/user/dashboard` and page where we utilize the `l10n` hook and make sure you are not getting any hydration errors.
